web/src/components/netflow-traffic.tsx

```diff
@@ -209,6 +221,9 @@ export const NetflowTraffic: React.FC<NetflowTrafficProps> = ({
         (isDNSTracking() || !fd.id.startsWith('dns_')) &&
         (isPktDrop() || !fd.id.startsWith('pkt_drop_')) &&
         (isFlowRTT() || fd.id !== 'time_flow_rtt') &&
+        (isUdn() || fd.id !== 'udns') &&
+        (isPktXlat() || fd.id.startsWith('xlat_')) &&
```

Comment:
Think this is the issue
```suggestion
        (isPktXlat() || !fd.id.startsWith('xlat_')) &&
```

Reply:
Oopsie ! Thanks mate !